Week03/lesson06/homework/startup_matchmaker/css/developersstyle.css

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
I'm wondering why you decided to have two separate stylesheets for the two files? The pages reuse much of the same CSS... But I respect that this may be less confusing for you